scripts/ci/config.yml

```diff
@@ -99,7 +99,8 @@ keras_tf2:
   <<: *tmpl
   override_name_or_path: test_keras_impl.py
   dependencies:
-    - "tensorflow==2.2.0"
+    - "protobuf<3.20,>=3.9.2"
+    - "tensorflow==2.7.3"
```

Comment:
I was thinking to lock this between a range, maybe 2.7.3 -> 2.9.1?

Reply:
It’s better to lock a version in testing

Reply:
also lock it to the lowest support version help us to discover problems. I didn't meet tensorflow problem until yesterday because I'm on the latest tensorflow version